index.html

```diff
@@ -1,286 +1,430 @@
+      </a>
+      <a
+        class="social-link"
+        href="https://www.linkedin.com/in/spayco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/linkedin.svg"
+          alt="LinkedIn Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://angel.co/spayco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/angel.svg"
+          alt="AngelList Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://twitter.com/iSpaYco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/twitter.svg"
+          alt="Twitter Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://spayco.medium.com/"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/png/xicon.png"
+          alt="Medium Icon"
+        />
+      </a>
+    </footer>
+    <div class="nav-overlay off-display">
+      <div class="nav-mobile-container">
+        <div class="xicon-image"><img src="/assets/icons/xicon.png" alt="Exit Icon"></div>
+        <ul class="overlay-menu d-flex">
+          <li class="overlay-link"><a href="#works">Portoflio</a></li>
+          <li class="overlay-link"><a href="#about">About</a></li>
+          <li class="overlay-link"><a href="#contact">Contact</a></li>
         </ul>
       </div>
-    </section>
-  </main>
-  <footer class="footer--section">
-    <div class="footer--section--icons"> 
-      <ul class="footer--social--icons"> 
-        <li>
-          <a href="https://github.com/joskalenda">
-            <img src="./assets/icons/github.png" alt="github account">
-          </a>
-        </li>
-        
-        <li>
-          <a href="https://www.linkedin.com/in/jos-kalenda-908069205">
-            <img src="./assets/icons/linkedin.png" alt="linkedin account">
-          </a>
-        </li>
-
-        <li>
-          <a href="**">
-            <img src="./assets/icons/peace.png" alt="peace">
-          </a>
-        </li>
-
-        <li>
-          <a href="https://www.twitter.com/in/joskalenda">
-            <img src="./assets/icons/twitter.png" alt="twitter account">
-          </a>
-        </li>
-        
-        <li>
-          <a href="#">
-            <img src="./assets/icons/medium.png" alt="meduim">
-          </a>
-        </li>
-      </ul>
     </div>
-    <ul class="mail">
-      <li>
-        <a href="#">joskalenda3@gmail.com</a>
-      </li> 
-    </ul>
-  </footer> 
-  <div class="nav-overlay off-display">
-    <div class="nav-mobile-container">
-      <div class="xicon-image"><img src="/assets/icons/xicon.png" alt="Exit Icon"></div>
-      <ul class="overlay-menu d-flex">
```

Comment:
- [ ]  Well that with your project, however the link you are using to the image does not exit. It could be better if you attach to the image an existing path to the image :+1: